filters/filters.go

```diff
@@ -14,6 +15,10 @@ import (
 	"github.com/zalando/skipper/skipper"
 )
 
+var (
```

Comment:
no need for the braces when there is only a single var, but of course fine like this, too